src/binance/accountbuilder.js

```diff
@@ -28,3 +28,30 @@ export async function getAccountBuilders() {
 
 	return Promise.all(stats);
 }
+
+
+export async function getAccountBuildersStreamable() {
```

Comment:
These are the same algorithms as the regular `getAccountBuilders` functions but they don't return `Promise.all()` so we don't have to wait